CNAME

```diff
@@ -1 +0,0 @@
-unitary.fund
```

Comment:
You'll need to leave this in place in order for the site's domain to continue to work with github pages.

Reply:
yes sorry, I was trying to render the github pages on my fork to preview it online (as I do [here]( https://nathanshammah.github.io/qutip.github.io/) with this [fork](https://github.com/nathanshammah/qutip.github.io)) but it's fine to look at the html locally only.